cli/internal/helm/client.go

```diff
@@ -45,7 +44,16 @@ func NewClient(kubeConfigPath, helmNamespace string, client *apiextensionsclient
 		return nil, fmt.Errorf("initializing config: %w", err)
 	}
 
-	return &Client{config: actionConfig, crdClient: client, log: log}, nil
+	kubeconfig, err := os.ReadFile(kubeConfigPath)
+	if err != nil {
+		return nil, fmt.Errorf("reading gce config: %w", err)
+	}
```

Comment:
nit: call `afero.NewOsFs()` before this block. This allows us to use `fs.ReadFile()` instead of having to use the `os` package to do so.